slack-base/src/test/resources/member_left_public_channel.json

```diff
@@ -0,0 +1,19 @@
+{
+  "token": "redacted",
+  "team_id": "redacted",
+  "api_app_id": "redacted",
+  "event": {
+    "type": "member_left_channel",
+    "user": "redacted",
+    "channel": "redacted",
+    "channel_type": "C",
+    "team": "redacted",
+    "event_ts": "1540822163.000100"
```

Comment:
I thought this event has no TS.

Reply:
`//Member joined events do not have a ts, so we manually set it as null`

Reply:
Discussed offline.
So, this event has it but it isn't explicitly stated in the documentation and it doesn't seem like anybody needs that timestamp so we'll proceed with current implementation but will leave this test JSON with `event_ts` since we receive it sometimes.